src/components/dev-hub/tab.js

```diff
@@ -28,10 +31,8 @@ const TabButton = styled('button')`
     transition: 0.3s;
     background-color: inherit;
     width: 136px;
-    padding-top: ${size.micro};
-    padding-bottom: ${size.medium};
-    padding-left: ${size.mediumLarge};
-    padding-right: ${size.mediumLarge};
+    padding: ${size.micro} ${size.medium} ${size.mediumLarge}
```

Comment:
Whoops just noticed this! With the padding shorthand it is actually `top right bottom left` so the correct use here would be `padding: ${size.micro} ${size.mediumLarge} ${size.medium} ${size.mediumLarge}`.

Even better there is a shorter shorthand which is `padding: <top> <left and right> <bottom>`, so `padding: ${size.micro} ${size.mediumLarge} ${size.medium}` is also equivalent!